src/epub/create_epub.py

```diff
@@ -0,0 +1,165 @@
+import os
+import zipfile
+
+from lxml import etree
+import shutil
+from zipfile import ZipFile
+from datetime import datetime
+
+import src.html.create_songs_html as cash
+
+
+class SongMeta:
+    def __init__(self, title='', alias='', plik=''):
+        self.title = title if title else ''
+        self.alias = alias if alias else ''
+        self.plik = plik
+
+    # def __repr__(self):
+    #     return "(%s, alias: %s, plik: %s)" % (self.title, self.alias, self.plik)
+
+    @staticmethod
+    def parseDOM(root, plik):
+        def elementTextOrNone(elem):
+            return elem.text if elem is not None else None
+        return SongMeta(
+            title=root.get('title'),
+            alias=elementTextOrNone(root.find('{*}alias')),
+            plik=plik
+        )
+
+
+def add_song(path, lista):
+    tree = etree.parse("../../songs/" + path + ".xml")
```

Comment:
Can we get full path as the argument here ? 

add_song("../../songs/foo.xml", lista) 

Would be the invocation. 
------------------------------------

Ideally I would see this as: 
   
    lista.append(newSong(""../../songs/foo.xml"));
    
as it's better to avoid mutable arguments.

Reply:
Done